<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>additiveClass</class>
 <widget class="QMainWindow" name="additiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>591</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Additive Widgets</string>
  </property>
  <property name="windowIcon">
   <iconset resource="additive.qrc">
    <normaloff>:/additive/icon.ico</normaloff>:/additive/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color: #FFFFFF;
	font: 10pt &quot;Lucida Sans&quot;;
	background-color: #505050;
	selection-background-color:#FEE728;
}

QPushButton {
color: white;
border: 1px solid #cccccc;
border-radius: 12px;
padding: 5px;
background-color:qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #777777, stop: 1 #222222);
min-width: 80px;
}

QPushButton:hover {
border: 1px solid #ffffff;
color:white;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #222222, stop: 1 #777777);
}

QPushButton:pressed {
color: white;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #000, stop: 1 #000);
}

QCheckBox, QRadioButton {	
 
	border-color: rgb(255, 255, 255);
	alternate-background-color: rgb(120, 255, 96);
}

QMenu  {
	selection-color: #000000;
	color: #000000;
	background-color: #FFFFFF;
}

QMenuBar {
	border-bottom: 1px solid #999955;
}

 QTabWidget::pane { 
     border-left: 1px solid #eee;
     border-top: 1px solid #eee;
     border-right: 1px solid #aaa;
     border-bottom: 1px solid #aaa;
 }

QLineEdit {
color: white;
selection-color: blue;
                        
} 

QStatusBar {
	color: #FFFFFF;
	border-top: 2px solid #000;
	background-color: #252525;


}

 QCheckBox::indicator
  {
	width: 18px;
	height: 18px;
  }
 QCheckBox::indicator:checked
  {
	image: url(:/additive/checked.gif);
  }
  QCheckBox::indicator:unchecked
  {
	image: url(:/additive/unchecked.gif);
  }


#centralWidget2 {
	border: 2px solid #FFC35B;
}

#qv{
	border: 2px solid #FFC35B;
}

QListWidget {
	
	selection-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHello">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Edge</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>22</width>
      <height>121</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Systems</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Skeletals</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Muscle</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Rendering</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Solid</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wireframe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Groups</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="centralWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>601</width>
      <height>371</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Opacity:</string>
    </property>
   </widget>
   <zorder>dial</zorder>
   <zorder>btnHello</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>tabWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label</zorder>
   <zorder>centralWidget2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>FullScreen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="additive.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFull_Screen</sender>
   <signal>triggered()</signal>
   <receiver>additiveClass</receiver>
   <slot>slot_fullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_fullScreen()</slot>
 </slots>
</ui>
